core/sql/executor/ExExplain.cpp

```diff
@@ -1724,9 +1724,9 @@ short ExExplainTcb::getExplainFromRepos(char * qid, Lng32 qidLen)
   if (vi->get(0, ptr, len))
     goto label_error2;
   
-  explainFragLen_ = str_decoded_len(len); // remove trailing null terminator
+  explainFragLen_ = str_decoded_len(len - 1); // remove trailing null terminator
```

Comment:
The JIRA isn't showing  trace fileor debugging info. Can you explain what the issue was with passingin 'len' into str_decoded_len ? Seems like the explainFragLen_ would have had sufficient length to allocate the explain fragment ? Some explanation in the PR or JIRA will help.

Reply:
for the '-1' , i think it should remove trailinng null terminator as the comment.
but last time in anoop 's commit, the '-1' was removed, but the method (getExplainFromRepos) is for EXPLAIN_QID,  not for EXPLAIN_STMT (see TRAFODION-1755), so i guess he changed the place there is no need to change.

Reply:
@sandhyasun, are you content with this change now? Can I merge it?